app/views/govuk_publishing_components/components/docs/title.yml

```diff
@@ -43,12 +43,12 @@ examples:
       average_title_length: long
   with_margin:
     description: |
-      Configurable bottom margin for the component. This has been built in anticipation of being compatible with a theoretical future model for component margins.
+      The component accepts a number for margin bottom from 0 to 9 (0px to 60px) using the [GOV.UK Frontend spacing scale](http://govuk-frontend-review.herokuapp.com/docs/#settings/spacing-variable-govuk-spacing-points). It defaults to having a margin bottom of 40px.
```

Comment:
Consider linking to the official design system documentation here, as the frontend review application is subject to change over time.

https://design-system.service.gov.uk/styles/spacing/#the-responsive-spacing-scale

Reply:
Excellent suggestion. I've now updated this in this yml file and all others that had this link. Thanks.